lib/lune.js

```diff
@@ -288,6 +300,25 @@ function phaseHunt (sdate) {
 }
 
 function phaseRange (start, end, phase) {
+  if (!(start && start instanceof Date)) {
+    throw new TypeError('First argument must be a Date object.')
+  }
```

Comment:
`if (!(start instanceof Date))` is sufficient, since every falsy value is not an instance of the Date class:

```
> false instanceof Date
false
> null instanceof Date
false
> undefined instanceof Date
false
> "" instanceof Date
false
> 0 instanceof Date
false
```
etc.

Reply:
right. I think my brain thought instanceof was a method for some reason